external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/EventHubSpout.java

```diff
@@ -152,7 +152,7 @@ public void open(Map config, TopologyContext context, SpoutOutputCollector colle
     try {
       preparePartitions(config, totalTasks, taskIndex, collector);
     } catch (Exception e) {
-      logger.error(e.getMessage());
+	  collector.reportError(e);
```

Comment:
It does log it, and puts it on the UI.

https://github.com/apache/storm/blob/28563ece16274bcd61827f38e960ce2d27a57dea/storm-core/src/jvm/org/apache/storm/executor/error/ReportError.java#L61

There is throttling around it so if we get too many errors too quickly they will not all be logged and not all of them will be written to ZK.